doc/installation.rst

```diff
@@ -21,19 +21,24 @@ Downloading and Installation
 Prerequisites
 ~~~~~~~~~~~~~~~
 
-The lmfit package requires `Python`_, `NumPy`_, `SciPy`_, `asteval`_, `six`_,
-and `uncertainties`_.
+Lmfit works with Python versions 2.7 and 3.4 - 3.7. Support for Python 2.6
+and 3.3 ended with lmfit versions 0.9.4 and 0.9.8, respectively.  Support
+for 2.7 is expected to end in early 2019.
 
-Lmfit works with Python versions 2.7 and 3.4 - 3.7. Support for Python 2.6 and
-3.3 ended with lmfit versions 0.9.4 and 0.9.8, respectively. Scipy version 0.17,
-NumPy version 1.10, asteval version 0.9.12, six version 1.10, and uncertainties
-version 3.0 are the minimum requirements.
+Lmfit requires the following Python packages, with versions given:
+   * `six` version 1.10 or higher.
+   * `NumPy` version 1.10 or higher.
+   * `SciPy` version 0.17 or higher.
+   * `asteval` version 0.9.12 or higher.
+   * `uncertainties` version 3.0 or higher.
```

Comment:
you lost the hyperlinks above for the packages, just add the ```_``` after the package name again.